api/app/signals/apps/users/change_log/tracker/user.py

```diff
@@ -0,0 +1,44 @@
+from django.contrib.gis.db import models
+
+from change_log.tracker import BaseChangeTracker
+
+
+class UserChangeTracker(BaseChangeTracker):
+    def changed_data(self):
+        """
+        Override the default ChangeTracker.
+
+        This custom ChangeTracker for Users is used to make sure the changes are stored correctly in the JSON
+        field (data) of the change_log.
+        """
+        _changed_data = dict()  # Here we are going to keep track of the changes
+
+        for field in self.instance._meta.get_fields():
```

Comment:
Seems a bit copy and pasty (compare to the `CategoryTracker`) --- not a show stopper though.